src/styles/App.css

```
@@ -0,0 +1,14 @@
+.App {
+	display: flex;
+	flex-direction: column;
+}
+.nasa-logo {
+	margin: auto;
+	width: 300px;
+	mix-blend-mode: darken;
+}
+h2 {
```

Good to leave clear line between different classes to make it easier to read
```suggestion
}

h2 {
```

Also a point about styling html elements directly: Generally speaking its better to avoid directly styling html elements (built in things like `h2`, form elements, list elements etc) directly, and instead apply a class with a useful name (eg `.heading`) to them, and apply your styles to that. This is best particularly for things like colour which is likely to change across a site.
If you have a set of font sizes for example you would like to use for all `hx` elements these would be better set in your main `index.css` file so they can apply to the entire site and make it easier to maintain going forward